### PR TITLE
Add kinetic tremor test

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
This cleans up the merging of https://github.com/Sage-Bionetworks/MotorControl-iOS/pull/10 back into master.

### CODE REVIEW

File: MotorControl/MotorControl/Resources/Kinetic_Tremor.json [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+                        "type": "videoView",
+                        "buttonTitle" : "See this in action",
+                        "url" : "instruction_video.mp4" },
+                 }
+             },
+             {
+                 "identifier":"eachHandInstructions",
+                 "type":"instruction",
+                 "title":"Each hand for 30 seconds",
+                 "text":"We will do this finger-to-nose motion repeatedly for 30 seconds with one hand, and do 30 seconds with your other hand.",
```
Comment (Erin-Mounts) [inline MotorControl/MotorControl/Resources/Kinetic_Tremor.json]:
```suggestion
                 "text":"We will do this finger-to-nose motion repeatedly for 30 seconds with one hand, and then do 30 seconds with your other hand.",
```
(?)

File: MotorControl/MotorControl/Resources/Kinetic_Tremor.json [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+                 }]
+             },
+             {
+                "identifier":"sitDownInstruction",
+                "type":"instruction",
+                "title":"Sit down",
+                "text":"Find a spot to sit where you can have your feet resting flat on the floor.",
+                "image":{
+                    "type":"animated",
+                    "imageNames":["FingerToNose0", "FingerToNose1"],
```
Comment (Erin-Mounts) [inline MotorControl/MotorControl/Resources/Kinetic_Tremor.json:95]: I'm guessing despite the image names, this animates sitting down with your feet flat on the floor, and not touching your finger to your nose...?

**Review by Erin-Mounts — APPROVED** [review]
one copyediting suggestion, one question, looks good

File: MotorControl/MotorControl/Resources/Kinetic_Tremor.json (hunk 2 above)
Comment (syoung-smallwisdom) [reply MotorControl/MotorControl/Resources/Kinetic_Tremor.json:95]: Confirmed "Yes"

**Pull request merged**